zingolib/src/data/mod.rs

```diff
@@ -0,0 +1 @@
+pub mod witness_trees;
```

Comment:
I think you can just put this in data.rs, and delete data/mod.rs.

Reply:
ok